modules/nw-dns-forward.adoc

```diff
@@ -42,7 +42,13 @@ spec:
     - foo.com
     forwardPlugin:
       policy: Random <3>
-      upstreams: <4>
+      transportConfig:
+        tls:
+          caBundle
```

Comment:
Found one minor thing here; `caBundle` should be `caBundle:` (with a semi-colon)